pkg/chart/chart.go

```diff
@@ -426,18 +427,20 @@ func (c *Chart) loadMember(filename string) (*ChartMember, error) {
 	return result, nil
 }
 
-type ChartContent struct {
-	Chartfile *Chartfile     `json:"chartfile"`
-	Members   []*ChartMember `json:"members"`
+//chartContent is abstraction for the contents of a chart
```

Comment:
also not being used outside of the package. I use the practice "always private unless it has to be public" just to be safe.